src/containers/BookingPassengers/index.js

```diff
@@ -9,15 +9,25 @@ import {
 } from '../../components';
 import {colors} from '../../library/constants';
 
-export const SelectPassengers = ({navigation}) => {
-  const nextScreenCast = type =>
+export const SelectPassengers = ({route: {params}, navigation}) => {
+  console.log('====================================');
```

Comment:
Delete the logs if you don't need them more

Reply:
Logs deleted